test/test_document.rb

```diff
@@ -44,6 +44,16 @@ def assert_equal_value(key, one, other)
       assert_equal "foo.bar", @document.data["whatever"]
     end
 
+    should "be able to jsonify its data" do
+      require 'json'
```

Comment:
In this case I would just add it to Jekyll's main require file because it's part of StdLib but if you want to be local I would require it at the top of formatters file you edited because it has no performance impact and doesn't slow anything down because most of the time it's required anyways by one thing or another.